.buildkite/release_pipeline.yaml

```diff
@@ -190,6 +247,12 @@ steps:
       Appveyor has generated the necessary artifacts and uploaded them
       to Builder and Bintray. Please ensure those artifacts have been
       uploaded before continuing with the release.
+  
+  - block: ":dango: :linux: :two: Build x86_64-linux-kernel2 Launcher"
+    prompt: |
+      x86_64-linux-kernel2 launcher will need to be built by hand.  If 
+      you are promoting launchers in the following step, please build 
+      and upload and x86_64-linux-kernel2 launcher before continuing.
```

Comment:
Is this because of a lack of kernel2 workers in Builder (which would normally be building this)?

(Adding or linking to explicit directions for building a kernel2 launcher would be good here.)

Also, why :dango:? 😄

Reply:
Correct.  I will add those steps to the Release documentation and link to them.

I was thinking about how good a dango would be at the time I was working on this. I am happy to report that that particular issue was resolved 😄  Also, three little launchers all lined up?(I just made that up, but I kinda like it)